deploy-manage/security/secure-settings.md

```diff
@@ -16,232 +15,177 @@ mapped_pages:
-
+::::{tab-item} ECK
+:sync: eck
 
-**Add secret values**
+In ECK, the operator simplifies secure settings configuration by relying on Kubernetes `Secrets`.
 
-Add keys and secret values to the keystore.
-
-1. [Log into the Cloud UI](/deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md).
-2. On the **Deployments** page, select your deployment.
-
-    Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
-
-3. From your deployment menu, select **Security**.
-4. Locate **{{es}} keystore** and select **Add settings**.
-5. On the **Create setting** window, select the secret **Type**.
-6. Configure the settings, then select **Save**.
-7. All the modifications to the non-reloadable keystore take effect only after restarting {{es}}. Reloadable keystore changes take effect after issuing a [reload_secure_settings](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) API request.
-
-
-
-**Delete secret values**
+Refer to [Configure secure settings on ECK](./k8s-secure-settings.md) for details and examples.
+::::
 
-When your keys and secret values are no longer needed, delete them from the keystore.
+::::{tab-item} Self-managed
+:sync: self-managed
 
-1. [Log into the Cloud UI](/deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md).
-2. On the **Deployments** page, select your deployment.
+In self-managed deployments, you're responsible for configuring and maintaining the Elasticsearch keystore on each node individually.
 
-    Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
+{{es}} provides the [`elasticsearch-keystore`](elasticsearch://reference/elasticsearch/command-line-tools/elasticsearch-keystore.md) command-line tool to help with this task. It allows you to list, add, remove, and update secure settings, as well as protect the keystore with a password if wanted.
 
-3. From your deployment menu, select **Security**.
-4. From the **Existing keystores** list, use the delete icon next to the **Setting Name** that you want to delete.
-5. On the **Confirm to delete** window, select **Confirm**.
-6. All modifications to the non-reloadable keystore take effect only after restarting {{es}}. Reloadable keystore changes take effect after issuing a [reload_secure_settings](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) API request.
+:::{important}
+Secure settings must be specified on every node, and must have the same values across the cluster to ensure consistent behavior.
 
-:::{dropdown} Using the API
+Changes to the keystore take effect only after restarting {{es}}, except for [reloadable settings](#reloadable-secure-settings) that can be refreshed using the API.
+:::
 
-**Steps**
+**Create the keystore**
 
-Create the keystore:
+To create the `elasticsearch.keystore`, use the `create` command:
```

Comment:
consider doing it this way - the next line after the command specifies the file that it creates. alternatively say "elasticsearch.keystore file" (but that is more confusing imo)

```suggestion
To create the Elasticsearch keystore, use the `create` command:
```